packages/terra-list/src/ListSection.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import SectionHeader from './ListSectionHeader';
+
+// Disable this lint rule to allow for the metaData prop
+/* eslint-disable react/forbid-prop-types */
```

Comment:
Can you disable the specific line rather than ignoring this whole block?

Reply:
Good call.

Reply:
Updated.

Reply:
I have tried that before myself, and sadly was never able to get it to work. I guess since it's a data structure instead of a line of code that it's hard to recognize the new-line flag eslint has.